docs/UserGuide.md

```diff
@@ -283,7 +280,7 @@ Example:
 ### 2.18. Complete or Make Uncomplete a Task : `mark`
 
 Marks a task to be complete or incomplete.
-Format: `mark [complete/uncomplete] INDEX`
+Format: `mark [complete/incomplete] INDEX`
 
 > * Using complete as a parameter will mark the selected task as complete.
 > * Using incomplete as a parameter will mark the selected task as incomplete.
```

Comment:
i'm fine with that.
Do you think update should have a field where we can change complete/incomplete status?